src/components/ui/widget.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ReactNode } from 'react';
+
+interface CardContent {
+  children: ReactNode;
+}
+
+const Widget = ({ children }: CardContent) => {
+  return <div className='w-[576px] rounded-xl bg-secondary'>{children}</div>;
```

Comment:
Maybe it's better to define the width by max-width, what are your thoughts, @whyleonardo?

Reply:
This worked properly, so i will go on with this adjustment.